docs/faststore/components/organisms/payment-methods.mdx

```diff
@@ -41,13 +41,19 @@ Follow the instructions in the [Importing FastStore UI component styles](https:/
 ```tsx
 export const flags = [
   { icon: { icon: 'Visa' }, alt: 'Visa' },
```

Comment:
**todo:** the props are missing in the prop section, I'm not sure if its a preview related issue. Can you pls check it? Thanks

<img width="1130" alt="image" src="https://github.com/user-attachments/assets/9774a710-4081-42a2-9451-0d6f2f42a0e6">